frontend/src/App.js

```diff
@@ -41,11 +41,14 @@ function App() {
                 <Route exact path="/about" component={About} />
                 <Route exact path="/quiz" component={Quiz} />
                 <Route exact path="/leaderboard" component={Leaderboard} />
+                <Route exact path="/UserHistory" component={UserHistory} />
               </Switch>
             </body>
-            <footer>
-              <Footer />
-            </footer>
+            <div className="app-footer">
+              <footer>
+                <Footer />
+              </footer>
+            </div>
```

Comment:
You've got a div outside a body there!

Reply:
maybe that's why the footer isnt playing ball....

Reply:
That was my fault sorry

Reply:
That bloody footer

Reply:
changed it

Reply:
Still needs a bit of a tweak